---
title: "Selected Research"
output: 
  html_document:
    toc: FALSE
---

```{r, include = FALSE}
library(here)
knitr::opts_chunk$set(
  #tidy = TRUE,
  #tidy.opts = list(width.cutoff = 95),
  warning = FALSE, 
  eval = FALSE,
  root.dir = here()
)
```

Select manuscripts are highlighted below.  
Links to all my research can be found at [google scholar](https://scholar.google.com/citations?user=eDbDGyIAAAAJ&hl=en) 

<br>
<br>

#### [**Multiscale integration of human and single-cell variations reveals unadjuvanted vaccine high responders are naturally adjuvanted**](https://www.medrxiv.org/content/10.1101/2023.03.20.23287474v1)  
Advances in multimodal single cell analysis can empower high-resolution dissection of human vaccination responses. The resulting data capture multiple layers of biological variations, including molecular and cellular states, vaccine formulations, inter- and intra-subject differences, and responses unfolding over time. Transforming such data into biological insight remains a major challenge. Here we present a systematic framework applied to multimodal single cell data obtained before and after influenza vaccination without adjuvants or pandemic H5N1 vaccination with the AS03 adjuvant. Our approach pinpoints responses shared across or unique to specific cell types and identifies adjuvant specific signatures, including pro-survival transcriptional states in B lymphocytes that emerged one day after vaccination. We also reveal that high antibody responders to the unadjuvanted vaccine have a distinct baseline involving a rewired network of cell type specific transcriptional states. Remarkably, the status of certain innate immune cells in this network in high responders of the unadjuvanted vaccine appear “naturally adjuvanted”: they resemble phenotypes induced early in the same cells only by vaccination with AS03. Furthermore, these cell subsets have elevated frequency in the blood at baseline and increased cell-intrinsic phospho-signaling responses after LPS stimulation ex vivo in high compared to low responders. Our findings identify how variation in the status of multiple immune cell types at baseline may drive robust differences in innate and adaptive responses to vaccination and thus open new avenues for vaccine development and immune response engineering in humans.

| Manuscript  | code to reproduce analysis |
| :-------------: | :-------------: |
|[ <img src="man/figures/paper.png" width="50" >](https://www.medrxiv.org/content/10.1101/2023.03.20.23287474v1) | [<img src="man/figures/github-mark.png"  width="50" >](https://niaid.github.io/fsc/)  |


<br>
<br>


#### [**Normalizing and denoising protein expression data from droplet-based single cell profiling**](https://www.nature.com/articles/s41467-022-29356-8)  
Multimodal single-cell profiling methods that measure protein expression with oligo-conjugated antibodies hold promise for comprehensive dissection of cellular heterogeneity, yet the resulting protein counts have substantial technical noise that can mask biological variations. Here we integrate experiments and computational analyses to reveal two major noise sources and develop a method called “dsb” (denoised and scaled by background) to normalize and denoise droplet-based protein expression data. We discover that protein-specific noise originates from unbound antibodies encapsulated during droplet generation; this noise can thus be accurately estimated and corrected by utilizing protein levels in empty droplets. We also find that isotype control antibodies and the background protein population average in each cell exhibit significant correlations across single cells, we thus use their shared variance to correct for cell-to-cell technical noise in each cell. We validate these findings by analyzing the performance of dsb in eight independent datasets spanning multiple technologies, including CITE-seq, ASAP-seq, and TEA-seq. Compared to existing normalization methods, our approach improves downstream analyses by better unmasking biologically meaningful cell populations. Our method is available as an open-source R package that interfaces easily with existing single cell software platforms such as Seurat, Bioconductor, and Scanpy and can be accessed at “dsb [https://cran.r-project.org/package=dsb]”.



| Manuscript  | code to reproduce analysis |
| :-------------: | :-------------: |
|[ <img src="man/figures/paper.png" width="50" >](https://www.nature.com/articles/s41467-022-29356-8) | [<img src="man/figures/github-mark.png"  width="50" >](https://github.com/niaid/dsb_manuscript)  |


<br>
<br>

#### [**Contrasting autoimmune and treatment effects reveals baseline set points of immune toxicity following checkpoint inhibitor treatment**](https://www.biorxiv.org/content/10.1101/2022.06.05.494592v1)  
Immune checkpoint inhibitors (ICIs) have changed the cancer treatment landscape, but severe immune-related adverse events (irAEs) can be life-threatening or fatal and may prohibit patients from receiving further ICI treatment. While the clinical features of irAEs are well documented, molecular signatures, predictive biomarkers, and mechanisms of impending irAEs are largely unknown. In addition, the markers and mechanisms of ICI-induced antitumor immunity often overlap with those for irAEs. It is thus critical to uncover signatures associated specifically with irAEs but not with antitumor immunity. To identify circulating immune cell states associated with irAEs, we applied multimodal single cell analysis (CITE-seq) to simultaneously measure the transcriptome and surface proteins from peripheral blood mononuclear cells (PBMCs) collected before and after treatment with an anti-PD-L1 antibody (avelumab) in patients with thymic cancers (thymic epithelial tumors). All patients had an antitumor response, yet a subset developed muscle autoimmunity (myositis), a potentially life-threatening irAE. Mixed-effect modeling disentangled cell type-specific transcriptional states associated with ICI treatment responses from those of irAEs to identify temporally stable pre-treatment immune set points associated with irAEs only. These pre-treatment baseline signatures of irAE developed post-avelumab irAEs reflect correlated transcriptional states of multiple innate and adaptive immune cell populations, including elevation of metabolic genes downstream of mTOR signaling in T-cell subsets. Together these findings suggest putative pre-treatment biomarkers for irAEs following ICI therapy in thymic cancer patients and raise the prospect of therapeutically dampening autoimmunity while sparing antitumor activity in cancer patients treated with ICIs. Together, pre-treatment biomarkers and interventional therapeutics could help mitigate treatment discontinuation and improve clinical outcomes.

| Manuscript  | code to reproduce analysis |
| :-------------: | :-------------: |
|[ <img src="man/figures/paper.png" width="50" >](https://www.biorxiv.org/content/10.1101/2022.06.05.494592v1) | [<img src="man/figures/github-mark.png"  width="50" >](https://github.com/niaid/irae_manuscript)  |


