<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>research</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"> </a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="./index.html">research</a>
</li>
<li>
  <a href="./research.html">publications</a>
</li>
<li>
  <a href="./dsb.ref.html">software</a>
</li>
<li>
  <a href="./posts.html">posts</a>
</li>
<li>
  <a href="./MPM_CV.pdf">CV</a>
</li>
<li>
  <a href="./contact.me.html">contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/MattPM">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">research</h1>

</div>


<br>
<center>
Matt Mulè PhD
</center>
<p><br></p>
<p>How can we improve clinical outcomes for patients by measuring and
understanding their immune systems? I focused on developing the concept
that multicellular networks form homeostatic human immune system
“setpoints” predictive of vaccination, infection and cancer
immunotherapy outcomes in my PhD at Cambridge and NIH with <a
href="https://medicine.yale.edu/profile/john-tsang/?tab=bio">John Tsang
(NIH - NIAID)</a> and <a
href="https://www.wehi.edu.au/researcher/ken-smith/">Ken Smith
(Cambridge)</a>. On the methodological side, we developed computational
approaches for denoising and statistical modeling new multimodal single
cell profiling methods such as CITE-seq.</p>
<p><u>I’m currently focused on advancing the field of personalized
clinical immunology on two major fronts: </u> 1. Investigating the basic
science of how molecular human immune subtypes emerge and control
response to cancer immunotherapy, vaccination and treatment of immune
mediated inflammatory diseases. 2. Development of personalized
therapeutic strategies for patients informed by low-cost global immune
system profiling in the clinical immunology laboratory.</p>
<br>
<center>
Selected work
</center>
<p><br></p>
<div
id="integrating-population-and-single-cell-variations-in-vaccine-responses-identifies-a-naturally-adjuvanted-human-immune-setpoint"
class="section level4">
<h4><a href="/man/pdf/MPM_immunity_2024.pdf"><strong>Integrating
population and single-cell variations in vaccine responses identifies a
naturally adjuvanted human immune setpoint</strong></a></h4>
<strong>Immunity (2024)</strong> Multimodal single-cell profiling
methods can capture immune cell variations unfolding over time at the
molecular, cellular, and population levels. Transforming these data into
biological insights remains challenging. Here, we introduce a framework
to integrate variations at the human population and single-cell levels
in vaccination responses. Comparing responses following AS03-adjuvanted
versus unadjuvanted influenza vaccines with CITE-seq revealed
AS03-specific early (day 1) response phenotypes, including a B cell
signature of elevated germinal center competition. A correlated network
of cell-type-specific transcriptional states defined the baseline immune
status associated with high antibody responders to the unadjuvanted
vaccine. Certain innate subsets in the network appeared ‘‘naturally
adjuvanted,’’ with transcriptional states resembling those induced
uniquely by AS03-adjuvanted vaccination. Consistently, CD14+ monocytes
from high responders at baseline had elevated phospho-signaling
responses to lipopolysaccharide stimulation. Our findings link baseline
immune setpoints to early vaccine responses, with positive implications
for adjuvant development and immune response engineering.
<div class="button-container">
<p><a href="/man/pdf/MPM_immunity_2024.pdf"> <button>PDF</button> </a>
<a href="https://niaid.github.io/fsc/"> <button><a
href="https://niaid.github.io/fsc/"><img src="man/figures/github-mark.png" width="20"></a>
code </button> </a>
<a href="https:/doi.org/10.1016/j.immuni.2024.04.009">
<button>DOI</button> </a>
<a href="/man/pdf/immunity_2024_editorial.pdf"> <button>Editorial
coverage</button> </a></p>
</div>
<div style="display: flex; justify-content: space-between;">
<p><img src="man/figures/immunity_cover_2024.png" alt="Image 2" style="width: 35%;margin-right: 10px;">
<img src="man/figures/immunity_2024.png" alt="Image 1" style="width: 35%; margin-left: 10px;"></p>
</div>
<p><br> <br>
<img align="right"  height = "500" style="margin-left: 15px;" src="man/figures/dsb_fig1.png" /></p>
</div>
<div
id="normalizing-and-denoising-protein-expression-data-from-droplet-based-single-cell-profiling"
class="section level4">
<h4><a
href="https://www.nature.com/articles/s41467-022-29356-8"><strong>Normalizing
and denoising protein expression data from droplet-based single cell
profiling</strong></a></h4>
<p><strong>Nature Communications (2022)</strong> Multimodal single-cell
profiling methods that measure protein expression with oligo-conjugated
antibodies hold promise for comprehensive dissection of cellular
heterogeneity, yet the resulting protein counts have substantial
technical noise that can mask biological variations. Here we integrate
experiments and computational analyses to reveal two major noise sources
and develop a method called “dsb” (denoised and scaled by background) to
normalize and denoise droplet-based protein expression data. We discover
that protein-specific noise originates from unbound antibodies
encapsulated during droplet generation; this noise can thus be
accurately estimated and corrected by utilizing protein levels in empty
droplets. We also find that isotype control antibodies and the
background protein population average in each cell exhibit significant
correlations across single cells, we thus use their shared variance to
correct for cell-to-cell technical noise in each cell. We validate these
findings by analyzing the performance of dsb in eight independent
datasets spanning multiple technologies, including CITE-seq, ASAP-seq,
and TEA-seq. Compared to existing normalization methods, our approach
improves downstream analyses by better unmasking biologically meaningful
cell populations. Our method is available as an open-source R package
that interfaces easily with existing single cell software platforms such
as Seurat, Bioconductor, and Scanpy and can be accessed at “dsb [<a
href="https://cran.r-project.org/package=dsb"
class="uri">https://cran.r-project.org/package=dsb</a>]”.</p>
<div class="button-container">
<p><a href="/man/pdf/dsb_paper_2022.pdf"> <button>PDF</button> </a>
<a href="https://github.com/niaid/dsb_manuscript"> <button><a
href="https://github.com/niaid/dsb_manuscript"><img src="man/figures/github-mark.png" width="20"></a>
code </button> </a>
<a href="https://www.nature.com/articles/s41467-022-29356-8">
<button>DOI</button> </a>
<a href="https://CRAN.R-project.org/package=dsb"> <button>R
Package</button> </a></p>
</div>
<p><br>
<img align="right" height = "600" style="margin-left: 15px;" src="man/figures/irae_diagram.png" /></p>
</div>
<div
id="contrasting-autoimmune-and-treatment-effects-reveals-baseline-set-points-of-immune-toxicity-following-checkpoint-inhibitor-treatment"
class="section level4">
<h4><a
href="https://www.biorxiv.org/content/10.1101/2022.06.05.494592v1"><strong>Contrasting
autoimmune and treatment effects reveals baseline set points of immune
toxicity following checkpoint inhibitor treatment</strong></a></h4>
<p><strong>BioRxiv (2022)</strong> Immune checkpoint inhibitors (ICIs)
have changed the cancer treatment landscape, but severe immune-related
adverse events (irAEs) can be life-threatening or fatal and may prohibit
patients from receiving further ICI treatment. While the clinical
features of irAEs are well documented, molecular signatures, predictive
biomarkers, and mechanisms of impending irAEs are largely unknown. In
addition, the markers and mechanisms of ICI-induced antitumor immunity
often overlap with those for irAEs. It is thus critical to uncover
signatures associated specifically with irAEs but not with antitumor
immunity. To identify circulating immune cell states associated with
irAEs, we applied multimodal single cell analysis (CITE-seq) to
simultaneously measure the transcriptome and surface proteins from
peripheral blood mononuclear cells (PBMCs) collected before and after
treatment with an anti-PD-L1 antibody (avelumab) in patients with thymic
cancers (thymic epithelial tumors). All patients had an antitumor
response, yet a subset developed muscle autoimmunity (myositis), a
potentially life-threatening irAE. Mixed-effect modeling disentangled
cell type-specific transcriptional states associated with ICI treatment
responses from those of irAEs to identify temporally stable
pre-treatment immune set points associated with irAEs only. These
pre-treatment baseline signatures of irAE developed post-avelumab irAEs
reflect correlated transcriptional states of multiple innate and
adaptive immune cell populations, including elevation of metabolic genes
downstream of mTOR signaling in T-cell subsets. Together these findings
suggest putative pre-treatment biomarkers for irAEs following ICI
therapy in thymic cancer patients and raise the prospect of
therapeutically dampening autoimmunity while sparing antitumor activity
in cancer patients treated with ICIs. Together, pre-treatment biomarkers
and interventional therapeutics could help mitigate treatment
discontinuation and improve clinical outcomes.</p>
<div class="button-container">
<p><a href="https://www.biorxiv.org/content/10.1101/2022.06.05.494592v1">
<button>PDF</button> </a>
<a href="https://github.com/niaid/irae_manuscript"> <button><a
href="https://github.com/niaid/irae_manuscript"><img src="man/figures/github-mark.png" width="20"></a>
code </button> </a>
<a href="https://www.biorxiv.org/content/10.1101/2022.06.05.494592v1">
<button>DOI</button> </a></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
